tools/unmicst/macros.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">labsyspharm/unmicst:@TOOL_VERSION@</container>
+            <!--
+            <requirement type="package" version="3.7">python</requirement>
+            <requirement type="package" version="1.15.0">tensorflow</requirement>
+            <requirement type="package" version="1.15.1">tensorflow-estimator</requirement>
+            <requirement type="package">cudnn</requirement>
+            <requirement type="package" version="10.0">cudatoolkit</requirement>
+            <requirement type="package" version="0.17.2">scikit-image</requirement>
+            <requirement type="package" version="1.4.1">scipy</requirement>
+            <requirement type="package" version="2020.7.24">tifffile</requirement>
+            <requirement type="package" version="2019.7.2">czifile</requirement>
+            <requirement type="package" version="3.2.3">nd2reader</requirement>
+            -->
+        </requirements>
+    </xml>
+
+    <xml name="version_cmd">
+        <version_command>echo @TOOL_VERSION@</version_command> 
```

Comment:
The purpose of this command is to really communicate to the user which tool was run, not which tool we wanted to run. So a --help would be better here if possible.